src/components/screens/Authenticator/modules/SignIn/SignIn.interactor.vue

```diff
@@ -69,7 +69,7 @@ export default {
 
     async onSocial({ email }) {
       if (SING_IN_CHALLENGE_TYPES.includes(this.challengeType)) {
-        this.$emit('sign-in', { email });
+        this.$emit('sign-in-social', { email });
```

Comment:
no need it to rename, just change to
`this.$emit('sign-in-social', { email, isSocial: true });` here
and in upper code
change 
`this.$emit('sign-in', { email, serverMode });` to 
`this.$emit('sign-in', { email, serverMode, isSocial: false });`

and for `SignUp.interactor.vue` change
`this.$emit('sign-up', { email, password, isSignUp: true });`
to
`this.$emit('sign-up', { email, password, isSignUp: true, isSocial: false });`